packages/lab/src/list-next/useList.ts

```diff
@@ -13,43 +14,58 @@ interface UseListProps {
    * If true, all items in list will be disabled.
    */
   disabled?: boolean;
-  /* Value for the uncontrolled version. */
+  /* Value for the controlled version. */
+  highlightedIndex?: number;
+  /* Value for the controlled version. */
   selected?: string;
-  /* Initial value for the uncontrolled version. */
+  /* Initial value for the controlled version. */
   defaultSelected?: string;
```

Comment:
I think the defaultSelected could still be used for uncontrolled version, is that true?